pyartifactory/objects/object.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, artifactory: AuthModel) -> None:
             self._artifactory.auth[0],
             self._artifactory.auth[1].get_secret_value(),
```

Comment:
I think this will break if `auth` is None

Reply:
Right

Reply:
Fixed